### ISSUE TITLE
Bug: APCs on Sublevel 2 refused to charge despite being powered.

### ISSUE THREAD

**Issue by Holdelta** [Yawn-Wider/YWPolarisVore#437]
#### Brief description of the issue
After a solar grub event, the SMES output of Archaeology and Mining on Sublevel 2 were at 0kW, and even after hotwiring Mining directly to the grid, certain APCs refused to charge, as demonstrated in the picture attached.

Replacing the wire did not help.

#### What you expected to happen
As long as there is power in the wire connected to the APC, the APC should charge.

#### What actually happened
The APC, even when connected to a powered wire, did not charge.

#### Steps to reproduce
Have not reproduced this issue yet, as it appears to be a very specific problem.  Potential triggers that are not yet proven to cause it are:
-Solar Grubs
-Misplaced wire(unlikely since this problem did not occur until later on in the shift)
-Setting up RCON(configured it late[~8 hours] into the round)
-Disabling breakers as a result of setting up the RCON

#### Code Revision
2020-01-08
93db394d21ee2235557976b9cbd5af907a0d5f1d 
 
#### Anything else you may wish to add:
![image](https://user-images.githubusercontent.com/7005665/72018006-baefb980-321b-11ea-9763-41c62fb46bd3.png)

**Comment by RadiantFlash7** [comment]
@Holdelta Gonna need some more information here.
did the APCs shower power connected? or none at all?
If so, odds are the SMES is not outputting enough power, theres still a grub somewhere in the subgrid, or
Actually. Your subgrid isn't even set up there. 1.1MW is the Engine output power, I believe.
Which would mean theres probably grubs on station still, OR the rest of the station is still trying to charge, and theres not enough power output to charge everything at once.

**Comment by Holdelta** [comment]
@RadiantFlash7 
Okay, now that the round is over, will close this issue.  I ghosted halfway through the round and saw that a grub was still active in mining.  I did end up hot-wiring mining to the main grid which is why that output was like that with engine + solars because I couldn't think of any other reason at the time.

If I am able to reproduce this in another situation, I will reopen, but this was opened with suggestion by Hawk during a round.
